shared/languages/zh-CN.ts

```diff
@@ -250,40 +250,40 @@ export const ZH_CN = {
    EiffelTowerDesc: "所有相邻炼钢厂，获得 +N 生产、存储以及劳动者乘数。 N 取决于其相邻炼钢厂数量",
    Electricity: "电能",
    Electrification: "电气化",
-   ElectrificationPowerRequired: "Power Required",
+   ElectrificationPowerRequired: "所需功率",
    ElectrificationStatus: "现状",
    ElectrificationStatusActive: "运转中",
    ElectrificationStatusNoPower: "无电能",
    ElectrificationStatusNotActive: "未运转",
    ElectrificationUpgrade: "解锁电气化。允许建筑消耗电能促进生产",
    Electrolysis: "电解作用",
-   Embassy: "Embassy",
-   EmpireValueFromBuildings: "From Buildings",
-   EmpireValueFromResources: "From Resources",
+   Embassy: "大使馆",
+   EmpireValueFromBuildings: "来自建筑物",
+   EmpireValueFromResources: "来自资源",
    EmptyTilePageBuildLastBuilding: "建造上一个建筑",
    EndConstruction: "取消建造",
    EndConstructionDescHTML: "当你取消建造，所有已被使用的资源<b>将不会返还</b>。",
    Engine: "发动机",
    Engineering: "工程学",
    Enlightenment: "启蒙运动",
-   Enrichment: "Enrichment",
+   Enrichment: "丰富",
    EstimatedTimeLeft: "预计剩余时间",
    Exploration: "勘探",
-   Explorer: "Explorer",
-   ExploreThisTile: "Send An Explorer",
+   Explorer: "探险家",
+   ExploreThisTile: "派遣探险家",
    ExploreThisTileHTML:
-      "An explorer will explore <b>this tile and its adjacent tiles</b>. Explorers are generated in %{name}. You have %{count} explorers left",
-   ExtraGreatPeople: "%{count} Extra Great People",
+      "探险家将探索 <b>此地块与相邻地块</b>. 探险家是在 %{name} 中生成. 您还有 %{count} 个探险家",
+   ExtraGreatPeople: "%{count} 额外伟人",
    ExtraGreatPeopleAtReborn: "重生时的额外伟人",
    Faith: "信仰",
    Farming: "农业",
    FavoriteBuildingAdd: "加入偏好",
    FavoriteBuildingEmptyToast: "你没有任何偏好建筑",
    FavoriteBuildingRemove: "移出偏好",
    Feudalism: "封建主义",
-   Fibonacci: "Fibonacci",
+   Fibonacci: "斐波那契",
    FibonacciDescV2:
-      "+%{idle} Science from Idle Workers. +%{busy} Science from Busy Workers. Fibonacci's permanent upgrade cost follows Fibonacci sequence",
+      "+%{idle} 闲置工人科学产出. +%{busy} 忙碌工人科学产出. 斐波那契的升级成本遵循斐波那奇序列,
```

Comment:
@insraq Don't know if you noticed, I'm sure you have, but this is missing a`"` at the end